code/modules/modular_computers/hardware/printer.dm

```diff
@@ -50,6 +50,42 @@
 		qdel(I)
 		stored_paper++
 		return TRUE
+	if(istype(I, /obj/item/paper_bin))
+		var/obj/item/paper_bin/bin = I
+		if(LAZYLEN(bin.papers))
+			if(stored_paper >= max_paper)
+				to_chat(user, span_warning("You try to feed \the [bin] into [src]'s paper recycler, but its paper bin is full!"))
+				return FALSE
+			/// Number of sheets we're adding
+			var/num_to_add = 0
+			/// Some goober put their manifesto in the paperbin, complain at the user if that's why nothing happened
+			var/rejected_sheets = FALSE
+			for(var/obj/item/paper/the_paper in bin.papers) // Search for the first blank sheet of paper, then toss it in
+				if(the_paper.info != "") // Uh oh, paper has words! 
+					rejected_sheets = TRUE
+					continue
+				if(istype(the_paper, /obj/item/paper/carbon)) // Add both the carbon, and the copy
```

Comment:
I don't understand this, what makes carbon copies special? I thought they were just a copy?

Reply:
Carbon copies are actually two sheets of paper stuck together. Currently, you can separate them and add each one individually, resulting in two sheets added to the bin. The var it checks for is to see if its been separated, and thus just one sheet.

Reply:
Ah, I see